Project-Descriptions-and-Plans/CVA6/CVA6-plan-approved-2023-2024.md

```diff
@@ -0,0 +1,420 @@
+| Verification    | Verification step 1: DVPlans                                                                                                   | 2023-Q3       | TSS (lead), 10xEngineers, MU-Electronics, ECC   |
+| Verification    | Verification step 1: CV32A60X is verified at programmer's view level                                                           | 2023-Q4       | TSS (lead), 10xEngineers, MU-Electronics, ECC   |
+| Verification    | Verification step 1: Create Spike tandem                                                                                       | 2023-Q3       | TSS (lead)                                      |
+| Verification    | Verification step 2: DVPlans                                                                                                   | 2024-Q3       | TSS (lead), 10xEngineers, MU-Electronics, ECC   |
+| Verification    | Verification step 2: CV32A60X is fully verified (programmer's view level + microarchitecture)                                  | 2024-Q4       | TSS (lead), 10xEngineers, MU-Electronics, ECC   |
+| Verification    | CV32A60X released, Project Freeze (PF) checklist                                                                               | 2024-Q4       | TSS (lead) and verification team                |
+| Software        | Linux Yocto (with BuildRoot and OpenSBI) available in an OpenHW repository                                                     | 2023-Q2       | TRT                                             |
+|                 | _For information: FreeRTOS project related to CVA6_                                                                            |               |                                                 |
+| Related project | Validate PMP features on CVA6 with applications on FreeRTOS                                                                    | November 2023 | ECC                                             |
+| Related project | Integrate updated release of FreeRTOS kernel into CVA6 FreeRTOS with device drivers implemented, upstream to main repo         | TBD           | ECC                                             |
+| Related project | Port the new FreeRTOS on CVA6-apu                                                                                              | TBD           | ECC                                             |
+
+### Project tracking and meetings
+
+The progress will be tracked in CVA6 meetings.
+
+The various activities (core, verification, software) are reported to the relevant task groups.
+
+The CVA6 usually meets every week, the agenda contains progress and technical topics.
+The meetings are well suited for East Coast, Europe and India timezones. Once a month, the meeting starts later to accomodate participants from the West Coast.
+
+In addition, the contributors to verification activities have a dedicated weekly meeting.
+
```

Comment:
Can you add the checklist from the latest PA template? https://github.com/openhwgroup/programs/blob/master/process/PlanApprovedTemplate.md

Reply:
Added at the end of the doc.